README.md

```diff
@@ -236,7 +236,7 @@ Note: Although RetDec now supports a system-wide installation ([#94](https://git
     sudo pkg install cmake python37 bison autotools
     ```
   * `cmake .. -DCMAKE_INSTALL_PREFIX=<path>`
-  * `make -jN` (`N` is the number of CPU cores to use for parallel build)
+  * `make -jN` (`N` is the number of  of processes to use for parallel build, typically number of cores + 1 gives fastest compilation time)
```

Comment:
Nitpick: One missed occurrence of `of  of`. Can you please also fix it before I merge this?

Reply:
Sorry, I was lazy and used the online github editor, I have learned my lesson :)